config/dockertests.json

```diff
@@ -6,8 +6,8 @@
     "paidInFull": true,
     "returnErrorToUser": true,
     "mediation": true,
-    "mediationPilot": false,
-    "directionsQuestionnaire": false,
+    "mediationPilot": true,
```

Comment:
why have you changed this to true?

Reply:
this is only relevant for local testing - the citizen-frontend in our integration-tests uses this config. we should prob only enable this when we enable in aat/prod.

Reply:
i shouldn't have changed mediationPilot, that's got nothing to do with what i am working on as part of this PR. So I put it back to false.

Reply:
we should revert this back to current prod config - but remember to enable here we we flick the switch